pkg/kubelet/cm/internal_container_lifecycle.go

```diff
@@ -54,19 +54,18 @@ func (i *internalContainerLifecycleImpl) PreStartContainer(pod *v1.Pod, containe
 }
 
 func (i *internalContainerLifecycleImpl) PreStopContainer(containerID string) error {
-	if i.cpuManager != nil {
-		return i.cpuManager.RemoveContainer(containerID)
-	}
+	// We can't safely call i.cpuManager.RemoveContainer(containerID)
+	// here. Regular containers could be in the process of restarting, and
+        // RemoveContainer() would remove any allocated exclusive CPUs that the
```

Comment:
Can you fix the indenting here?

Reply:
oops...editor was set to use spaces instead of tabs.